lisa/testsuite.py

```diff
@@ -49,6 +51,9 @@ def __init__(
         else:
             self.tags = []
         self.description = description
+        if requirement is None:
+            requirement = DEFAULT_REQUIREMENT
```

Comment:
Could this just be the default value instead of it being Optional and None?

Reply:
fixed.